lib/npmdc/checker.rb

```diff
@@ -36,7 +37,12 @@ def call
         raise(MissedDependencyError, dependencies: @missing_dependencies)
       end
 
-      output("Checked #{@dependencies_count} packages. Everything is ok.", :success)
+      result_stats = "Checked #{@dependencies_count} packages. " +
+                     "Warnings: #{@suspicious_dependencies.count}. " +
+                     "Errors: #{@missing_dependencies.count}. " +
```

Comment:
Yep. Alos, It would be better to have message like this even for the failed checks